tests/src/Kernel/ContentRenderTestBase.php

```diff
@@ -0,0 +1,242 @@
+use Drupal\user\Entity\Role;
+use Drupal\user\RoleInterface;
+
+/**
+ * Base class for testing the content being rendered.
+ */
+abstract class ContentRenderTestBase extends MultilingualAbstractKernelTestBase {
+
+  use SparqlConnectionTrait;
+
+  /**
+   * The node storage.
+   *
+   * @var \Drupal\node\NodeStorageInterface
+   */
+  protected $nodeStorage;
+
+  /**
+   * The node view builder.
+   *
+   * @var \Drupal\Core\Entity\EntityViewBuilderInterface
+   */
+  protected $nodeViewBuilder;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'address',
+    'field',
+    'field_group',
+    'link',
+    'file',
+    'text',
+    'typed_link',
+    'maxlength',
+    'entity_reference',
+    'entity_reference_revisions',
+    'composite_reference',
+    'inline_entity_form',
+    'datetime',
+    'datetime_range',
+    'node',
+    'media',
+    'views',
+    'entity_browser',
+    'media_avportal',
+    'media_avportal_mock',
+    'filter',
+    'oe_media',
+    'oe_media_avportal',
+    'oe_content',
+    'oe_content_entity',
+    'oe_content_entity_contact',
+    'oe_content_entity_organisation',
+    'oe_content_extra',
+    'oe_content_extra_project',
+    'oe_content_timeline_field',
+    'oe_content_news',
+    'oe_content_page',
+    'oe_content_policy',
+    'oe_content_departments_field',
+    'oe_content_documents_field',
+    'oe_content_publication',
+    'oe_content_reference_code_field',
+    'oe_content_featured_media_field',
+    'oe_content_project',
+    'oe_whitelabel_extra_project',
+    'sparql_entity_storage',
+    'rdf_skos',
+    'file_link',
+    'options',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->setUpSparql();
+
+    $this->installEntitySchema('node');
+    $this->installSchema('file', 'file_usage');
+    $this->installSchema('node', ['node_access']);
+    $this->installEntitySchema('media');
+    $this->installEntitySchema('file');
+
+    $this->installConfig([
+      'file',
+      'field',
+      'entity_reference',
+      'entity_reference_revisions',
+      'composite_reference',
+      'node',
+      'media',
+      'filter',
+      'oe_media',
+      'media_avportal',
+      'oe_media_avportal',
+      'typed_link',
+      'address',
+    ]);
+
+    // Importing of configs which related to media av_portal output.
+    $this->container->get('config.installer')->installDefaultConfig('theme', 'oe_whitelabel');
+
+    $this->container->get('module_handler')->loadInclude('oe_content_documents_field', 'install');
+    oe_content_documents_field_install(FALSE);
+
+    $this->installConfig([
+      'oe_content',
+      'oe_content_entity',
+      'oe_content_entity_contact',
+      'oe_content_entity_organisation',
+      'oe_content_timeline_field',
+      'oe_content_departments_field',
+      'oe_content_reference_code_field',
+      'oe_content_featured_media_field',
+      'oe_content_news',
+      'oe_content_page',
```

Comment:
remove these modules that we dont use
oe_content_page
oe_content_news
oe_content_policy
oe_content_publication
etc

from the entire file

Reply:
the etc included also oe_content_entity_contact, double check pls what u need & what not